libsql/sql.go

```diff
@@ -23,15 +23,29 @@ type LibsqlDriver struct {
 
 // ExtractJwt extracts the JWT from the URL and removes it from the url.
 func extractJwt(query *url.Values) (string, error) {
-	authToken := query.Get("authToken")
+	authTokenSnake := query.Get("auth_token")
+	authTokenCamel := query.Get("authToken")
 	jwt := query.Get("jwt")
+	query.Del("auth_token")
 	query.Del("authToken")
 	query.Del("jwt")
-	if authToken != "" && jwt != "" {
-		return "", fmt.Errorf("both authToken and jwt are present in the url. Please use only one of them")
+
+	nonempty := func(x string) int {
```

Comment:
Maybe `count` would better describe what this function is doing?

Reply:
this is pretty confusing that a function which seems to have bool result returns integer.
Maybe it would be better to have `countNonEmpty(authTokenSnake, authTokenCamel, jwt) > 1` and inside `countNonEmpty` have an int counter that you increment for each non empty value like:
```
func countNonEmpty(slice ...string) int {
	count := 0
	for _, s := range slice {
		if s != "" {
			count++
		}
	}
	return count
}
```

Reply:
That's indeed much better, thank you :) Fixed